datahub/investment/project/urls.py

```diff
@@ -77,6 +79,11 @@
         audit_item,
         name='audit-item',
     ),
+    path(
+        'investment/<uuid:pk>/update-stage',
+        update_stage_of_item,
+        name='update_stage_of_item',
```

Comment:
Would be good to use hyphens in the name to keep this consistent with other routes.
```suggestion
        name='update-stage-of-item',
```